src/OrleansTestingHost/AppDomainSiloHandle.cs

```diff
@@ -58,7 +58,7 @@ public static SiloHandle Create(string siloName, Silo.SiloType type, ClusterConf
                     optimizer.AddCachedAssembly(assembly.Key, assembly.Value);
                 }
 
-                var args = new object[] { siloName, type, config };
+                var args = new object[] { siloName, siloBuilderFactory, config };
```

Comment:
I'm not sure I understand now how the SiloType is being used... In the past that was what defined whether the Primary silo has the membership and reminder grains, but even though that approach was not very good, I'm not understanding how you are achieving that now. Can you clarify?

Reply:
My understanding is that primary is now keyed off of the silo name, so the 'type' is no longer necessary.
@ReubenBond, is this correct?